draft-packages/empty-state/package.json

```diff
@@ -35,10 +35,10 @@
     "@kaizen/component-library": "^13.2.4",
     "@kaizen/draft-illustration": "^4.1.5",
     "@types/classnames": "^2.3.0",
-    "classnames": "^2.3.1"
+    "classnames": "^2.3.1",
+    "@kaizen/component-base": "^1.1.0"
```

Comment:
Move above `"@kaizen/component-library"` since it autogenerates in alphabetical order (to prevent unnecessary diffs for later package changes)